docs/architecture.md

```diff
@@ -3,33 +3,47 @@
 ## Domain Driven Design
 
 We follow the principles of domain driven design.<br>
-Business logic is handled in the domain layer.
+**Business logic is handled in the domain layer.**
 
-Business logic is mainly handled within an Aggregate. If that is not possible, e.g. because an operation affects multiple aggregates a Domain Service is used.<br>
-A new Aggregate is created by a factory.<br>
-To load an Aggregate from the persistence layer a Repository is used.<br>
-A changed Aggregate is persisted via a Repository as well.<br>
+### Aggregates, Factories, Repositories ###
+Business logic is mainly handled within an *aggregate*. If that is not possible, e.g. because an operation affects multiple *aggregates* a domain *service* is used.<br>
```

Comment:
s/an operation/a domain operation/ It's all easy going when you come at your problem from the outside